gordo_components/builder/build_model.py

```diff
@@ -516,7 +528,6 @@ def cache_key(self) -> str:
                 "name": self.name,
                 "model_config": self.model_config,
                 "data_config": self.data_config,
-                "user-defined": self.metadata,
```

Comment:
Should also remove the  `if self.metadata is None:`... above since it was only there because of this line you now removed.

Reply:
Nice catch! :baseball: